chia/wallet/wallet_transaction_store.py

```diff
@@ -291,7 +305,14 @@ async def get_not_sent(self) -> List[TransactionRecord]:
         records = []
         for row in rows:
             record = TransactionRecord.from_bytes(row[0])
-            if record.name in self.tx_submitted:
+            if self.last_global_tx_resend_time < current_time - self.global_tx_resend_timeout_secs:
+                # Reset the "sent" state for peers that have replied about this transaction. Retain errors.
+                record = dataclasses.replace(record, sent=1, sent_to=filter_ok_mempool_status(record.sent_to))
+                await self.add_transaction_record(record, False)
+                self.tx_submitted[record.name] = current_time, 1
+                records.append(record)
+                self.last_global_tx_resend_time = current_time
```

Comment:
If there are no rows, we will never reset the self.last_global_tx_resend time, and will keep calling this method over and over. Shouldn't we reset it at the beginning of the method?

Reply:
Thanks!